### PR TITLE
added a filter feature with ascending or descending options

### PR DESCRIPTION
**Pull request by Jay-R-Mac**

### CODE REVIEW

File: src/components/BrowseArticles.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { dateChanger } from "../../utils/utils";
+
+export default function BrowseArticles({ sortBy, sortAsc }) {
+  const { topic } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [articles, setArticlesInfo] = useState([]);
+
+  function filterSort(articles, sortBy, sortAsc) {
+    const sortedArticles = articles.slice().sort((a, b) => {
```
Comment (Augs0) [inline src/components/BrowseArticles.jsx:14]: You shouldn't need to sort manually here. Remember that your api should accept sort by queries which we can access when making the api call

File: src/components/BrowseArticles.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      if (sortAsc === "Desc") {
+        if (sortByA > sortByB) return -1;
+        if (sortByA < sortByB) return 1;
+      } else {
+        if (sortByA < sortByB) return -1;
+        if (sortByA > sortByB) return 1;
+      }
+      return 0;
+    });
+    console.log(sortedArticles);
```
Comment (Augs0) [inline src/components/BrowseArticles.jsx]: Don't forget to remove unnecessary console logs

**Review by Augs0 — CHANGES_REQUESTED** [review]
Hey Jordan, this is looking really good but the implementation needs a bit of work. The main thing I want you to change is the sorting. This shouldn't need to be done manually as your api call should accept sort by queries. Take a look back at your BE repo if you need a reminder as to how that looks. Also, I noticed that although all the other sort queries work, your date sorting does not seem to work. No matter if I click `asc` or `desc` for the `date` option, the same order seems to show

File: src/components/CommentDeleter.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// import { deleteArticleComments } from "./Axios";
+import { useEffect, useState, useContext } from "react";
+
+
+
+export default function CommentDeleter({comment_id, commentAuthor}) {
+  const [deleteComment, setDeleteComment] = useState(false);
+
+
+//   useEffect(() => {
```
Comment (B7Bprog) [inline src/components/CommentDeleter.jsx:10]: Please remove all commented-out code.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Jordan, please check my other comment, otherwise very nice work, it all works fine.

**Pull request merged**